Src/Web/Web.Shared.Net/Implementation/RequestTrackingExtensions.cs

```diff
@@ -6,7 +6,11 @@
     using System.Web;
     using Microsoft.ApplicationInsights.Common;
     using Microsoft.ApplicationInsights.DataContracts;
-    using Microsoft.ApplicationInsights.W3C;
+#if DEPENDENCY_COLLECTOR
```

Comment:
RequestTrackingExtensions should never be compiled into the dependency collector, so `using Microsoft.ApplicationInsights.W3C.Internal` is the only possible option